app/jsx/pages/PageBase.jsx

```diff
@@ -183,6 +184,19 @@ class PageBase extends React.Component
     })
   }
 
+  // External Google Analytics trackers
+  runExtGATracker = (tracker, id) => {
+    ReactGA.ga('create', id, 'auto', {'name': tracker})
+    ReactGA.ga(tracker+'.send', 'pageview', {'page': window.location.pathname})
+  }
+
+  extGA = (unit_id) => {
+    // ToDo: Turn on at launch
+    // if (/jmie_sfews/.test(unit_id)) { this.runExtGATracker('sfewsTracker', 'UA-31540406-1') }
+    // if (/^nanocad/.test(unit_id)) { this.runExtGATracker('nanocadTracker', 'UA-17962781-1') }
+    // if (/^uciem_westjem/.test(unit_id)) { this.runExtGATracker('westjemTracker', 'UA-34762732-1') }
+  }
+
```

Comment:
Ultimately I'd like this driven from the database, but I realize we just have to jam it in for now.